crates/oxc_prettier/src/printer/mod.rs

```diff
@@ -60,6 +60,8 @@ impl<'a> Printer<'a> {
                 Doc::Softline => self.handle_softline(indent, mode),
                 Doc::Hardline => self.handle_hardline(indent),
                 Doc::IfBreak(doc) => self.handle_if_break(doc.unbox(), indent, mode),
+                // TODO: need refactor `fits` to accept `cmd` instead of `doc`.
+                Doc::Fill(_) => {}
```

Comment:
Seems we need to accept `cmd` by parameters in `fits`🤔. @Dunqing 

https://github.com/prettier/prettier/blob/eebf0e4b5ec8ac24393c56ced4b4819d4c551f31/src/document/printer.js#L470-L477

Reply:
In prettier's fits implementation, only mode and doc are used. so we don't need cmd. i think that's enough!

Reply:
I changed to accept `Command` finally for `handle_fill`. 😥